en/SUMMARY.md

```diff
@@ -388,10 +388,13 @@
   - [Helicopter (experimental)](frames_helicopter/index.md)
     - [Helicopter Config/Tuning](config_heli/index.md)
   - [Rovers (experimental)](frames_rover/index.md)
-    - [Drive Modes](flight_modes_rover/index.md)
-    - [Differential-steering Rover](frames_rover/differential_rover.md)
+    - [Differential Rover](frames_rover/differential.md)
+      - [Drive Modes)](flight_modes_rover/differential.md)
```

Comment:
Note, reordered drive modes first. Also removed the suffixes "DR" here. I don't think we need it here. The page itself does identify the type of drive mode.